libs/island-ui/core/src/lib/Select/Select.tsx

```diff
@@ -40,6 +40,7 @@ export const Select = <
   errorMessage = '',
   placeholder = '',
   defaultValue,
+  hideSelectedOptions = true,
```

Comment:
Won't defaulting to `true` here change the behaviour of every usage of Select?

Reply:
Yes that's true, it defaults as true anyways in the React Select component so this wouldn't change anything but is unnecessary so we will remove it, don't remember why I put it there in the first place.